globus_sdk/transfer/client.py

```diff
@@ -334,6 +334,67 @@ def submit_delete(self, data):
         """
         return self.post('/delete', data)
 
+    def make_submit_transfer_data(self, source_endpoint, dest_endpoint,
+                                  transfer_items, label, sync_level=None,
+                                  **kwargs):
```

Comment:
I disagree very strongly with trying to do an extra level of verification in the sdk. The REST APIs already should be doing verification and returning useful errors. Maintaining yet another level of verification is a waste of resources. In a user interface, it may be necessary, but I don't see how it is in an sdk.